docs/RtspStreamingGuide.md

```diff
@@ -0,0 +1,371 @@
+# USB Camera Device Service RTSP Streaming Guide
+
+## Contents
+
+[System Requirements](#system-requirements)  
+[How It Works](#how-it-works)  
+[Overview](#overview)  
+[Tested Devices](#tested-devices))  
+[Dependencies](#dependencies)  
+[Get the Source Code](#get-the-source-code)
+[Configuration options](#configuration-options)
+[Deploy the Service](#deploy-edgex-and-usb-device-camera-microservice)  
+[Verify the Service](#verify-service-and-device-profiles)   
+[Adding Devices using REST API](#adding-devices-using-rest-api)
+[Start Video Streaming](#start-video-streaming)
+[Shutting Down](#shutting-down)  
+[Troubleshooting](#troubleshooting)) 
+[License](#license)
+
+## System Requirements
+
+- Intel&#8482; Core&#174; processor
+- Ubuntu 20.04.4 LTS
+- USB-compliant Camera
+
+**Time to Complete**
+
+10-20 minutes
+
+**Other Requirements**
+
+You must have administrator (sudo) privileges to execute the user guide commands.
+
+## How It Works
+For an explanation of the architecture, see the [Readme](../README.md#how-it-works).
+
+## Overview
+EdgeX device service for communicating with USB cameras attached to Linux OS platforms.
+This service provides the following capabilities:
+- V4L2 API to get camera metadata.
+- Camera status
+- Video stream reference
+- FFmpeg framework to capture video frames and stream them to an RTSP server.
+- An [RTSP server](https://github.com/aler9/rtsp-simple-server) is embedded in the dockerized device service. 
+
+## Tested Devices
+The following devices have been tested with EdgeX:
```

Comment:
Maybe ".. have been test to work with the EdgeX USB Camera Device Service." Just "EdgeX" seems kind of generic. Additionally, maybe a note that YMMV depending on hardware/firmware version and operating system support. Since it's all a moving target.